ProcessAPI/ComputeEngine.java

```diff
@@ -0,0 +1,21 @@
+
+public class ComputeEngine implements ProcessInterface, ProcessResponse{
+
+    @Override
+    public ProcessResponse processData(Sources input, Sources output) {
+        // TODO Auto-generated method stub
+        throw new UnsupportedOperationException("Unimplemented method 'processData'");
+    }
+
+    @Override
+    public boolean isValid() {

```

Comment:
Im not sure if she'll accept a boolean here

Reply:
Ideally it would be wrapped in an enum =)